goship.go

```diff
@@ -760,23 +777,210 @@ func extractOutputHandler(fn func(http.ResponseWriter, *http.Request, string, st
 	}
 }
 
+type User struct {
+	UserName, UserAvatar string
+	auth                 auth
+}
+
+func getUser(r *http.Request) (User, error) {
+	u := User{}
+	session, err := store.Get(r, session_name)
```

Comment:
If you can, could this variable (`session_name`) be renamed to `sessionName` to follow Go conventions?